.github/workflows/python-deploy.yml

```
@@ -62,13 +62,13 @@ jobs:
           (. .venv/install/${{ env.BIN_DIR }}/activate && python -m pip install --no-index --find-links=partcad-cli/dist partcad-cli && deactivate)
       # Upload artifact from the selected OS/Python version combination only
       - name: Upload "partcad"
-        if: "matrix.os == 'ubuntu-24.04' && matrix.python-version == '3.11'"
+        if: 'matrix.os == "ubuntu-24.04" && matrix.python-version == "3.11"'
```

actionlint (1.7.4)

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)